src/metrics.rs

```diff
@@ -50,16 +51,77 @@ pub fn export_metrics(
 
     for (source, scan) in scans {
         for entry in scan.matches {
-            let source = source.clone().into();
+            let source: SourceLabels = source.clone().into();
+            let title: String = format!(
+                "{} {}: {}",
+                source.image.clone().unwrap_or_default(),
+                entry.artifact.name,
+                entry.vulnerability.id
+            );
+            let (cvss_base_score, cvss_exploitability_score, cvss_impact_score) =
+                if !entry.vulnerability.cvss.is_empty() {
+                    (
+                        entry
+                            .vulnerability
+                            .cvss
+                            .first()
+                            .unwrap_or_else(|| {
+                                panic!(
+                                    "Unable to get cvss_base_score from {:?}",
+                                    entry.vulnerability.cvss
+                                )
+                            })
+                            .metrics
+                            .base_score
+                            .to_string(),
+                        entry
+                            .vulnerability
+                            .cvss
+                            .first()
+                            .unwrap_or_else(|| {
+                                panic!(
+                                    "Unable to get cvss_base_score from {:?}",
+                                    entry.vulnerability.cvss
+                                )
+                            })
+                            .metrics
+                            .exploitability_score
+                            .to_string(),
+                        entry
+                            .vulnerability
+                            .cvss
+                            .first()
+                            .unwrap_or_else(|| {
+                                panic!(
```

Comment:
I don't think we should panic here and just write a fallback-value instead

Reply:
good point, done.